stdlib/contrib/anaisdg/anomalydetection/mad.flux

```diff
@@ -0,0 +1,40 @@
+package anomalydetection 
+
+import "math"
+import "experimental"
+
+mad = (table=<-, threshold) => {
+    // _value_med = med(x)
+    data = table |> group(columns: ["_time"], mode:"by")
+    med = data |> median(column: "_value")
+    
+    // _value_diff = xi-med(xi)
+    diff = experimental.join(
+    left: data,
+    right: med,
+    fn: (left, right) => ({ right with _value: math.abs(x: left._value - right._value) })
+    )
+   
+   //The constant b is needed to make the estimator consistent for the parameter of interest.
+    /// In the case of the usual parameter a at Gaussian distributions
+    b = 1.4826
```

Comment:
```
// The constant b makes the estimator consistent for the parameter of interest.
// In the case of the usual parameter at Gaussian distributions
b = 1.4826
```

I'm not sure what that 2nd line in the comment is trying to say, but it seems like an incomplete sentence.

Reply:
The complete sentence is "In the case of the usual parameter at Gaussian distributions
b = 1.4826"
Do you have a suggestion for how I should change it? Does this read better? 
"In the case of the usual parameter at Gaussian distributions, " 
b = 1.4826
Or should I just do: 
"In the case of the usual parameter at Gaussian distributions b = 1.4826" 
Thank you

Reply:
Oh I see. The variable declaration is meant to be read as a continuation of the comment. I think that's what threw me. I'm used to comments being complete ideas without relying on the accompanying source code. I don't know that it's wrong; just not something I'm used to.